src/uuid.ts

```diff
@@ -0,0 +1,51 @@
+const UUIDPattern = /^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i;
+
+export function formatUUID(uuid: string): string {
```

Comment:
should the `formatUUID` and `parseUUID` methods actually be exported? I think we would at least want to mark them `@internal` for now.

Reply:
just `parseUUID` does 👍